ssl-certificate-update.md

```diff
@@ -152,7 +150,7 @@ Steps are simple:
 If you've read the previous sections, you will know what this means (and
 shame on you if you have not).
 
-We need to download [AddTrustExternalCARoot-2048.pem](https://raw.githubusercontent.com/rubygems/rubygems/master/lib/rubygems/ssl_certs/AddTrustExternalCARoot-2048.pem).
+We need to download [AddTrustExternalCARoot.pem](https://raw.githubusercontent.com/rubygems/rubygems/master/lib/rubygems/ssl_certs/rubygems.org/AddTrustExternalCARoot.pem).
```

Comment:
Perhaps a side note for people updating 2.6.x?
> ** Trying to update 2.6.x? **
Download [GlobalSignRootCA.pem](https://raw.githubusercontent.com/rubygems/rubygems/master/lib/rubygems/ssl_certs/index.rubygems.org/GlobalSignRootCA.pem) as well